decidim-admin/app/views/decidim/admin/static_pages/_form.html.erb

```diff
@@ -0,0 +1,13 @@
+<div class="field">
+  <%= form.translated :text_field, :title, autofocus: true %>
+</div>
+
+<% if can? :update_slug, form.object %>
+  <div class="field">
+    <%= form.text_field :slug %>
+  </div>
+<% end %>
+
+<div class="field">
+  <%= form.translated :text_area, :content %>
```

Comment:
We need to change this when https://github.com/AjuntamentdeBarcelona/decidim/pull/227 is merged (NO HURRIES @beagleknight)